local-cli/server/server.js

```diff
@@ -12,6 +12,7 @@ const chalk = require('chalk');
 const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
+const findSymlinksPaths = require('./findSymlinksPaths');
```

Comment:
Yes. But on the other hand there is no way to know if you're using symlinks or not.